webpack/assets/javascripts/react_app/common/colors.scss

```diff
@@ -7,7 +7,7 @@ $pf-gold-400: #f0ab00;
 $pf-green-400: #3f9c35;
 $pf-green-600: #1e4f18;
 $pf-blue-400: #0088ce;
-$pf-color-white: #fff;
+$pf-v5-color-white: #fff;
```

Comment:
Bring back to pf-color-white (was cought automatically by `pf-c` -> `pf-v5-c` replace)